bedrock/settings/base.py

```diff
@@ -241,10 +241,11 @@ def lazy_langs():
 
 # TODO create these files by porting from their .lang equivalents
 FLUENT_DEFAULT_FILES = ['brands', 'navigation', 'footer', 'download_button']
```

Comment:
I know it may not apply since these files will come after the active language file for a page, but would it still make sense to exclude these files from activation status / metadata creation?

Reply:
I thought about that. I didn't do that yet because I thought perhaps having an updated list of the locales that had met the 80% rule might be a nice thing to know. It would be easy to exclude them though if that info isn't needed.

Reply:
Ok yeah true, let’s leave it as-is for now